### DISCUSSION TITLE
How to achieve relative rotation?

### DISCUSSION THREAD

**Discussion by Yutaka-Morita** [bevyengine/bevy#10159] [Q&A]
I want the rotation of a child object to be relative to the rotation of its parent object.
```
* parent_transform = Transform::from_rotation(
  Quat::from_rotation_z(self.tau) * (
    Quat::from_rotation_y(self.phi) * Quat::from_rotation_x(self.theta)
  )
);
```
When the child object tries to look up (rotate on the x-coordinate axis) relative to this parent, it behaves as expected.
```
child_transform = Transform::from_rotation(Quat::from_rotation_x(self.theta_diff));
// (after this the rotation axis of the parent is reset)
```
However, when a rotation in another coordinate axis is given, the rotation axis will be such that the parent rotation is not taken into account.
How can we achieve rotation relative to any axis of rotation?

**Comment by Yutaka-Morita** [comment]
I could have solved it myself.

**Reply by NiklasEi** [reply]
It would be nice if you could document your solution for others finding this question and mark it as answered by your own response. :slightly_smiling_face:
